packages/form-core/tests/FormApi.spec.ts

```
@@ -366,6 +366,92 @@ describe('form api', () => {
+    const form = new FormApi({
+      defaultValues: {
+        people: [
+          {
+            firstName: '',
+            lastName: '',
+          },
+        ],
+      },
+      validators: {
+        onMount: ({ value }) => {
+          let fieldErrors: Record<string, string> = {}
```

ESLint

[error] 'fieldErrors' is never reassigned. Use 'const' instead.

(prefer-const)